oemof/solph/components.py

```diff
@@ -477,17 +479,17 @@ class GenericInvestmentStorageBlock(SimpleBlock):
           (InvestmentFlow.invest(n, target(n)) + existing) *
           invest\_relation\_input_output(n) \\
           \forall n \in \textrm{INVEST\_REL\_IN\_OUT}
-
-    Maximal capacity :attr:`om.InvestmentStorage.max_capacity[n, t]`
-        .. math:: E(n, t) \leq invest(n) \cdot c_{min}(n, t), \\
-            \forall n \in \textrm{MAX\_INVESTSTORAGES,} \\
+    # TODO:
```

Comment:
What does that mean?

Reply:
Good question. That was there already. I will check the blame.

Reply:
Ah, no. That was me :) I guess that I should check that again.

Reply:
I compared the docstring with the constraint defined below. Seems consistent. I removed the TODOs.